datastore/src/main/resources/application.properties

```diff
@@ -6,5 +6,5 @@ spring.datasource.password=
 #spring.jpa.show-sql=true
 spring.jpa.hibernate.ddl-auto=update
 application.baseUrl=${ADDRESS}
-#server.port= ${PORT}
+server.port= ${PORT}
```

Comment:
I think its best if we remove the space before the `$` on this line and the next